src/main/kotlin/com/hibob/academy/employee_feedback_feature/service/FeedbackService.kt

```diff
@@ -1,13 +1,17 @@
 package com.hibob.academy.employee_feedback_feature.service
 
 import com.hibob.academy.employee_feedback_feature.dao.*
+import com.hibob.academy.employee_feedback_feature.validation.ValidateSubmission.Companion.validateTextSubmission
 import org.springframework.beans.factory.annotation.Autowired
 
 import org.springframework.stereotype.Component
 
 @Component
 class FeedbackService @Autowired constructor(private val feedbackDao: FeedbackDao) {
     fun submitFeedback(feedbackSubmission: FeedbackSubmission): Long {
+        validateTextSubmission(feedbackSubmission.feedbackText, 300)
+        validateTextSubmission(feedbackSubmission.department, 100)
```

Comment:
do we really want to do this validation on department?

NOT IN THE SCOPE - but theoretically, the would choose a department from a closed list of existing departments in the company. This wouldn't be a free text